magento/magento2-page-builder
# Remove modalLoaded = false before opening media browser

### Depeneds On
https://github.com/magento/magento2/pull/27566

### Related Pull Requests
https://github.com/magento/magento2/pull/27566
https://github.com/magento/adobe-stock-integration/pull/1163


### Description (*)


After merge https://github.com/magento/magento2/pull/27566 this property no longer actual
As media Gallery now reloads dynamically.
this was added by https://github.com/magento/magento2/pull/23621 now it properly fixed by  https://github.com/magento/magento2/pull/27566 
### Story
n/a

### Bug
https://github.com/magento/adobe-stock-integration/issues/935

### Task
n/a

### Fixed Issues (if relevant)

1. magento/magento2-page-builder#<issue_number>: Issue title

### Builds


### Manual testing scenarios (*)

1. Open PageBuilder on diferent pages like CMS create page or Category create Page
2. Add multiple rows with image widget
3. Click select from gallery multiple times, from multiple rows
4. Verify that images inserted correctly to each instance.
5. Verify that gallery opened correctly and updated correctly

### Questions or comments


### Checklist
